wsd/LOOLWSD.cpp

```diff
@@ -4254,21 +4254,25 @@ int LOOLWSD::getClientPortNumber()
     return ClientPortNumber;
 }
 
-std::vector<int> LOOLWSD::getKitPids()
+std::map<pid_t, pid_t> LOOLWSD::getKitPids()
 {
-    std::vector<int> pids;
+    std::map<pid_t, pid_t> pids;
+    pid_t pid;
     {
         std::unique_lock<std::mutex> lock(NewChildrenMutex);
         for (const auto &child : NewChildren)
-            pids.push_back(child->getPid());
+        {
+            pid = child->getPid();
+            pids.emplace(pid, pid);
```

Comment:
std::set ? 

std::map is for mapping, but since both key and value are the same, std::set is more fitting.

Reply:
> std::map is for mapping, but since both key and value are the same, std::set is more fitting.

You're right. I'll replace map wit set.